Header header
geometry_msgs/Point pt_1
geometry_msgs/Point pt_2
geometry_msgs/Point pt_3
geometry_msgs/Point pt_4
geometry_msgs/Point pt_5
geometry_msgs/Point pt_6
int64 landmark_check

